CSharp/BatchExplorer/Views/MainView.xaml

```diff
@@ -561,100 +561,69 @@
+                                    <MenuItem Header="Disable" Command="{Binding SelectedJobSchedule.DisableJobSchedule}" CommandParameter="{Binding SelectedJobSchedule}" />
+                                    <MenuItem Header="Delete" Command="{Binding SelectedJobSchedule.Delete}" CommandParameter="{Binding SelectedJobSchedule}" />
                                 </ContextMenu>
                             </DataGrid.ContextMenu>
                             <DataGrid.Columns>
-                                <DataGridTextColumn Header="Id" Binding="{Binding Id}"/>
-                                <DataGridTextColumn Header="Display Name" Binding="{Binding DisplayName}"/>
+                                <DataGridTextColumn Header="Id" Binding="{Binding Id}" />
+                                <DataGridTextColumn Header="Display Name" Binding="{Binding DisplayName}" />
                                 <DataGridTextColumn Header="State" Binding="{Binding State}">
                                     <DataGridTextColumn.CellStyle>
                                         <Style TargetType="DataGridCell">
-                                            <Setter Property="Foreground" Value="{Binding State, Converter={StaticResource JobScheduleStateColorConverter}}"/>
-                                            <Setter Property="FontWeight" Value="Bold"/>
-                                            <Setter Property="Background" Value="White"/>
+                                            <Setter Property="Foreground" Value="{Binding State, Converter={StaticResource JobScheduleStateColorConverter}}" />
+                                            <Setter Property="FontWeight" Value="Bold" />
+                                            <Setter Property="Background" Value="White" />
                                         </Style>
                                     </DataGridTextColumn.CellStyle>
                                 </DataGridTextColumn>
-
-                                <DataGridTextColumn Header="" Width="*" CanUserReorder="False" CanUserSort="False"/>
+                                <DataGridTextColumn Header="" Width="*" CanUserReorder="False" CanUserSort="False" />
                             </DataGrid.Columns>
                         </DataGrid>
                     </Grid>
-
                 </TabItem>
                 <TabItem Header="{Binding JobTabTitle}" IsSelected="{Binding JobTabIsSelected}">
                     <Grid>
                         <Grid.RowDefinitions>
                             <RowDefinition Height="Auto"/>
                             <RowDefinition Height="*"/>
                         </Grid.RowDefinitions>
-                        <StackPanel
-                            Grid.Row="0"
-                            Orientation="Horizontal">
+                        <DockPanel LastChildFill="True">
                             <Button

```

Comment:
VS Intellisense doesn't like Grid.Row = 0 under DockPanel. I was using StackPanel (what was their originally but couldn't get Search box visible or even to use up rest of space. A search of StackOverflow said to use DockPanel, which I did have better luck with. If you can get this to work with StackPanel and ideally a auto-resizing SearchBox then I would be open to changing it back.